tsconfig.json

```diff
@@ -14,6 +14,7 @@
     "examples/*/node_modules/**",
     "packages/*/node_modules/**",
     "extensions/*/node_modules/**",
```

Comment:
```suggestion
    "**/node_modules/**",
```
BTW, any idea why this is not `**/node_modules/**,` ?

Reply:
IDK, to be honest. I was thinking about simplifying the pattern too, but then decided to keep my changes minimal to get them landed faster.

Would you like to open a follow-up pull request to simplify this part of `exclude` configuration?